stdlib/Markdown/src/parse/parse.jl

```diff
@@ -1,5 +1,11 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    MD
+
+`MD` represents a Markdown document. However, `MD` objects should only be constructed using 
+the exported macros [`@md_str`](@ref) and [`@doc_str`](@ref).
+"""
 mutable struct MD
```

Comment:
We should also have a `public MD` statement if this is a part of the documented API.